internal/network/command_peering_list.go

```diff
@@ -0,0 +1,84 @@
+package network
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	networkingv1 "github.com/confluentinc/ccloud-sdk-go-v2/networking/v1"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/errors"
+	"github.com/confluentinc/cli/v3/pkg/output"
+)
+
+func (c *peeringCommand) newPeeringListCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "list",
+		Short: "Display peering connections in the current environment.",
+		Args:  cobra.NoArgs,
+		RunE:  c.list,
+	}
+
+	pcmd.AddContextFlag(cmd, c.CLICommand)
+	pcmd.AddEnvironmentFlag(cmd, c.AuthenticatedCLICommand)
+	pcmd.AddOutputFlag(cmd)
+
+	return cmd
+}
+
+func (c *peeringCommand) list(cmd *cobra.Command, _ []string) error {
+	peerings, err := c.getPeerings()
+	if err != nil {
+		return err
+	}
+
+	list := output.NewList(cmd)
+	for _, peering := range peerings {
+		if peering.Spec == nil {
+			return fmt.Errorf(errors.CorruptedNetworkResponseErrorMsg, "spec")
+		}
+		if peering.Status == nil {
+			return fmt.Errorf(errors.CorruptedNetworkResponseErrorMsg, "status")
+		}
+
+		cloud, err := getCloud(peering)
+		if err != nil {
+			return err
+		}
+
+		if output.GetFormat(cmd) == output.Human {
+			list.Add(&peeringHumanOut{
+				Id:        peering.GetId(),
+				Name:      peering.Spec.GetDisplayName(),
+				NetworkId: peering.Spec.Network.GetId(),
+				Cloud:     cloud,
+				Phase:     peering.Status.GetPhase(),
```

Comment:
@ejagade @kabraham13 
For the `list` command, I think it makes sense to output only non cloud specific values, otherwise the output would take up too much horizontal screen space.

For the `describe` command, I am going to output the following cloud specific values
For `AWS` -  `account`, `vpc`, `routes`, `customer_region`
For `GCP` - `project`, `vpc_network`
For `Azure` - `tenant`, `vnet`, `customer_region`

What do you guys think?

Reply:
One option I thought is - for each row print oneof the following, as a peering record can belong to only one cloud

oneof {
{awsAccount, awsVPC, region}, //aws
{gcpProject, vpc_network} //gcp
{azureTenant, azureVnet, region} //azure
}
this would add at most 3 columns to the output

But a customer can have with multiple peering records in more than 1 cloud. this would mean `list` command will contain 1 set of columns for each cloud

I'm ok with adding these in describe command